static/css/components/cbox.less

```diff
@@ -50,10 +48,12 @@
     cursor: pointer;
   }
   &Content {
+    width: 660px !important;
```

Comment:
I think we learned we can do `box-sizing: content-box;` here instead of any `!important` and resolve the issue